app/services/course/assessment/question/scribing_import_service.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+# Imports new pdf files, splits and processes the files and creates scribing questions for each
+# page of the PDF file.
+class Course::Assessment::Question::ScribingImportService
+  # Creates a new service import object.
+  #
+  # @param [Course::Assessment::Question::Scribing] question The scribing question for import.
+  def initialize(question, file)
+    @question = question
+    @file = file
+  end
+
+  # Imports and saves the provided PDF as a scribing question.
+  #
+  # @return [Boolean] True if the pdf is processed and successfully saved, otherwise false. Note
+  #   that if the save is unsuccessful, all questions are not persisted.
+  def save
+    return_value = true
+    Course::Assessment::Question::Scribing.transaction do
+      generate_pdf_files.each do |file|
+        question = build_scribing_question
```

Comment:
Not very understand why the service takes a question as the param and creates a new question here instead, it's very confusing because the user would expect the question given to be saved. 

I'd think it's better to remove the question from param and let the service to take only params from the controller and return a created question.